packages/lumberaxe/lumberaxe.gemspec

```
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", ">= 6.0.6.1", "< 7.1"
+  spec.add_dependency "activesupport", "7.1.3"
```

This library should not force a specific Rails version on its users.

```suggestion
  spec.add_dependency "activesupport", ">= 6.0.6.1", "<= 7.1"
```